src/store/elementalChat.js

```diff
@@ -278,10 +292,25 @@ export default {
     },
     refreshChatter ({ dispatch, rootState }) {
       callZome(dispatch, rootState, 'chat', 'refresh_chatter', null, 30000)
-        .catch(error => log('refreshChatter zome error', error))
     },
     joinChannel ({ commit }, payload) {
       commit('setCurrentChannelId', payload)
+    },
+    updateProfile ({ commit, dispatch, rootState }, payload) {
+      const args = {
+        nickname: payload,
+        avatar_url: ''
```

Comment:
`avatar_url` is an option so you don't need to pass it. unless you would like it to be a placeholder for now.